profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_core/tests/multisite_business_indicators_core.test

```diff
@@ -9,9 +9,12 @@
  */
 class BusinessIndicatorsCoreQAWebTestCase extends DrupalWebTestCase {
   protected $profile = 'multisite_drupal_communities';
-  
-  var $public_files_directory = NULL;
-  
+
+  protected $publicFilesDirectory = NULL;
```

Comment:
Change it back to `$public_files_directory`, or it won't work. This is a property of the parent method.